diff.go

```diff
@@ -53,12 +53,25 @@ const (
 	// DiffInfoComponentExposeUpdated
 	DiffInfoComponentExposeUpdated DiffType = "component-expose-updated"
 
-	// DiffInfoComponentScaleUpdated
-	DiffInfoComponentScaleUpdated DiffType = "component-scale-updated"
+	// scale
+
+	// DiffInfoComponentScalePlacementUpdated
+	DiffInfoComponentScalePlacementUpdated DiffType = "component-scale-placement-updated"
+
+	// DiffInfoComponentScaleUp
+	DiffInfoComponentScaleUp DiffType = "component-scale-up"
```

Comment:
Not sure this is always sufficient.

What about the following situation:

- component has scale min: 1, max: 3
- component has 3 instances

now you submit a new definition that has scale min: 1, max: 2
this should result in a scale-down also.